protostar/commands/shared_args.py

```diff
@@ -0,0 +1,9 @@
+from protostar.cli.command import Command
+
+output_shared_argument = Command.Argument(
```

Comment:
I think it's unnecessary, it's good kind of redundancy

Reply:
It's a bad redundancy because it makes sure these arguments can be configured in shared_command_configs.

Reply:
At the other hand, `output` can be misleading for deploy command.

Reply:
So we should do it for all of the redundant arguments I guess? can you make task for that?

Reply:
https://github.com/software-mansion/protostar/issues/294